### DISCUSSION TITLE
adding-client-side-validation-to-aspnet-core-without-jquery-or-unobtrusive-validation/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#128] [Announcements]
# adding-client-side-validation-to-aspnet-core-without-jquery-or-unobtrusive-validation/

In this article I describe how to use the aspnet-client-validation library to provide client-side validation instead of relying on jQuery.

https://andrewlock.net/adding-client-side-validation-to-aspnet-core-without-jquery-or-unobtrusive-validation/

**Comment by TorreyGarland** [comment]
Is this compatible with FluentValidation?

**Reply by TorreyGarland** [reply]
please disregard....

**Comment by bpkinez** [comment]
Excellent post, I didn't know for this alternative validation approach.

Thanks!

**Comment by haxxornulled** [comment]
I am coding up a FormManager Library that is almost finished,  written in TypeScript that includes an unobtrusive ASP.Net validation plugin. It is quite functional and very fast. The only dependency it has at the moment is Inversify which I use as my DI container. I will send you the GitHub repo when I finish.
